cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -321,358 +325,252 @@ describe('Credentials', () => {
-          cy.getByDataCy('label-credential-type').contains('Credential type');
-          cy.getByDataCy('credential-type').contains('Amazon Web Services');
-          cy.getByDataCy('label-access-key').contains('Access Key');
-          cy.getByDataCy('access-key').contains('username');
-          cy.getByDataCy('label-secret-key').contains('Secret Key');
-          cy.getByDataCy('secret-key').contains('Encrypted');
-          cy.getByDataCy('label-sts-token').contains('STS Token');
-          cy.getByDataCy('label-organization').contains('Organization');
-          cy.getByDataCy('organization').contains(awxOrganization.name);
-          cy.getByDataCy('label-description').contains('Description');
-          cy.getByDataCy('description').contains('description');
-          cy.getByDataCy('edit-credential').click();
-          cy.verifyPageTitle('Edit Credential');
-          const ModifiedCredentialName = credentialName + ' - edited';
-          cy.getByDataCy('name').type(ModifiedCredentialName);
-          cy.get('input[data-cy="username"]').then(($username) => {
-            cy.wrap($username).should('have.value', 'username');
-          });
-          cy.get('input[data-cy="password"]').then(($pwd) => {
-            cy.wrap($pwd).should('have.value', 'ENCRYPTED');
-          });
-          cy.get('input[data-cy="security-token"]').then(($token) => {
-            cy.wrap($token).should('have.value', 'ENCRYPTED');
-          });
-          const newDescription = 'new description';
-          cy.getByDataCy('description').clear().type(newDescription);
-          cy.clickButton(/^Save credential$/);
-          cy.getByDataCy('name').contains(ModifiedCredentialName);
-          cy.getByDataCy('label-credential-type').contains('Credential type');
-          cy.getByDataCy('credential-type').contains('Amazon Web Services');
-          cy.getByDataCy('label-access-key').contains('Access Key');
-          cy.getByDataCy('access-key').contains('username');
-          cy.getByDataCy('label-secret-key').contains('Secret Key');
-          cy.getByDataCy('secret-key').contains('Encrypted');
-          cy.getByDataCy('label-sts-token').contains('STS Token');
-          cy.getByDataCy('sts-token').contains('Encrypted');
-          cy.getByDataCy('label-description').contains('Description');
-          cy.getByDataCy('description').contains(newDescription);
-          cy.clickPageAction('delete-credential');
-          cy.get('#confirm').click();
-          cy.intercept('DELETE', awxAPI`/credentials/${newCred.id.toString()}/`).as('deleted');
-          cy.clickButton(/^Delete credential/);
-          cy.wait('@deleted')
-            .its('response')
-            .then((deleted) => {
-              expect(deleted?.statusCode).to.eql(204);
-              cy.verifyPageTitle('Credentials');
-            });
-        });
-    });
-
-    it.skip('can edit credential type that renders a sub form', () => {
```

Comment:
Why remove this one? 
This test is part of https://issues.redhat.com/browse/AAP-26426

Reply:
@himdel let me verify if I missed something here. I had several versions, and files to compare.

Reply:
It was removed before. Checkout the commit `187e79d45406a2ea46549bd5b88fc9812d2abd26` before the https://github.com/ansible/ansible-ui/pull/2618 was merged.